portal-api/api-tests/v1/users/post-create-user.api-test.js

```diff
@@ -133,8 +144,19 @@ describe('a user', () => {
         await createUser(newUser);
         const { status, body } = await as(aNonAdmin).get(BASE_URL);
 
-        expect(status).toEqual(200);
-        expect(body.users.find((user) => user.username === MOCK_USER.username).roles).toStrictEqual([READ_ONLY]);
+        expect(status).toEqual(403);
+        expect(body.error).toEqual('Unauthorized');
+        if(body.users && body.users.length > 0){
+          const user1 = body.users.find((user) => user.username === MOCK_USER.username);
+          if(user1){
+            expect(user1.roles).toStrictEqual([READ_ONLY, READ_ONLY]);
+          } else {
+            throw new Error(`User ${MOCK_USER.username} not found`);
+          }
+        } else {
+          expect(body.users).toBeUndefined();
+        }
+        // expect(body.users.find((user) => user.username === MOCK_USER.username).roles).toStrictEqual([READ_ONLY]);
```

Comment:
Same comment as the test above - this test is also testing `POST /users` and doesn't really care about the fact that `GET /users` returns a `403` for non-admin users.

Reply:
Changed as above comment